src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.scss

```diff
@@ -24,6 +24,14 @@
   height: 25px;
 }
 
+cds-header-item {
+  width: 600px;
```

Comment:
Probably this needs to be max-width otherwise not looking good.


![image](https://github.com/ceph/ceph/assets/6695156/3843b080-ee92-4def-ae17-d4ea31df796e)